tabs/src/components/sample/QueryRunner.jsx

```diff
@@ -6,12 +6,21 @@ import { gridCellWithFocusableElementBehavior, } from '@fluentui/accessibility';
 import { TrashCanIcon } from '@fluentui/react-icons-northstar';
 import { useTranslation } from "react-i18next";
 import "./style/QueryRunner.css";
+import PropTypes from 'prop-types';
 
-export function QueryRunner() {
+QueryRunner.propTypes = {
+    query: PropTypes.string,
+    setQuery: PropTypes.func
+};
+
+export function QueryRunner(props) {
 
     // Translations
     const { t } = useTranslation();
 
+    var query = props.query;
+    var setQuery = props.setQuery;
+
```

Comment:
change to `const`

Reply:
18b949a8384f973b0f17fdaf8bdfad4c52261944